 Fri, 04 Oct 2019 21:21:11  INFO logger.py <module> [ line:30 ] hehehe
 Fri, 04 Oct 2019 21:21:28  INFO basepage.py wait_ele_visible [ line:40 ] 等待 登录页_等待用户名元素可见  元素  ('id', 'com.cashier.jiutongshanghu:id/ed_login_phone') 可见成功。耗时：0:00:01.571432
 Fri, 04 Oct 2019 21:21:29  INFO basepage.py wait_ele_visible [ line:40 ] 等待 登录页_输入用户名  元素  ('id', 'com.cashier.jiutongshanghu:id/ed_login_phone') 可见成功。耗时：0:00:00.115708
 Fri, 04 Oct 2019 21:21:29  INFO basepage.py get_element [ line:77 ] 查找  登录页_输入用户名 元素 ('id', 'com.cashier.jiutongshanghu:id/ed_login_phone') 成功！
 Fri, 04 Oct 2019 21:21:29  INFO basepage.py input_text [ line:114 ] 在 登录页_输入用户名 元素 ('id', 'com.cashier.jiutongshanghu:id/ed_login_phone') 上输入文本值：13072721092 成功！
 Fri, 04 Oct 2019 21:21:31  INFO basepage.py wait_ele_visible [ line:40 ] 等待 登录页_输入密码  元素  ('id', 'com.cashier.jiutongshanghu:id/ed_login_password') 可见成功。耗时：0:00:01.554248
 Fri, 04 Oct 2019 21:21:31  INFO basepage.py get_element [ line:77 ] 查找  登录页_输入密码 元素 ('id', 'com.cashier.jiutongshanghu:id/ed_login_password') 成功！
 Fri, 04 Oct 2019 21:21:32  INFO basepage.py input_text [ line:114 ] 在 登录页_输入密码 元素 ('id', 'com.cashier.jiutongshanghu:id/ed_login_password') 上输入文本值：123456 成功！
 Fri, 04 Oct 2019 21:21:32  INFO basepage.py wait_ele_visible [ line:40 ] 等待 登录页_点击登录按钮  元素  ('id', 'com.cashier.jiutongshanghu:id/but_login') 可见成功。耗时：0:00:00.586944
 Fri, 04 Oct 2019 21:21:32  INFO basepage.py get_element [ line:77 ] 查找  登录页_点击登录按钮 元素 ('id', 'com.cashier.jiutongshanghu:id/but_login') 成功！
 Fri, 04 Oct 2019 21:21:32  INFO basepage.py click_element [ line:101 ] 点击  登录页_点击登录按钮 元素 ('id', 'com.cashier.jiutongshanghu:id/but_login') 成功！
 Fri, 04 Oct 2019 21:21:32  ERROR basepage.py wait_ele_exists [ line:49 ] 等待元素 ('-android uiautomator', 'new UiSelector().text("账单")') 存在 失败！
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\app_framework\common\basepage.py", line 46, in wait_ele_exists
    WebDriverWait(self.driver, timeout, frequency).until(ec.presence_of_element_located(loc))
  File "C:\Python37\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "C:\Python37\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 64, in __call__
    return _find_element(driver, self.locator)
  File "C:\Python37\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
AttributeError: 'str' object has no attribute 'find_element'
 Fri, 04 Oct 2019 21:21:32  ERROR basepage.py save_img [ line:166 ] 网页截图失败！
Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\app_framework\common\basepage.py", line 46, in wait_ele_exists
    WebDriverWait(self.driver, timeout, frequency).until(ec.presence_of_element_located(loc))
  File "C:\Python37\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "C:\Python37\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 64, in __call__
    return _find_element(driver, self.locator)
  File "C:\Python37\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
AttributeError: 'str' object has no attribute 'find_element'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\keen\PycharmProjects\app_framework\common\basepage.py", line 164, in save_img
    self.driver.save_screenshot(img_path)
AttributeError: 'str' object has no attribute 'save_screenshot'
